src/styles/globals.css

```diff
@@ -17,14 +17,17 @@ body {
 }
 
 @layer utilities {
+
```

Comment:
?

Reply:
local prettier updated the global.css reverted change